### PR TITLE
Add RBAC file watcher and alias tests

### PR DESCRIPTION
**Pull request by josephca**
## Description

Add e2e tests for new RBAC features

- load RBAC policy and conditional permission  from files 
- test aliases : $currentUser, $ownerRefs 

## Which issue(s) does this PR fix

- Fixes [RHIDP-3482](https://issues.redhat.com/browse/RHIDP-3482)

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Comment by josephca** [comment]
@subhashkhileri @gustavolira please review when you have a chance. The newly added tests have passed, and I'm trying to pass all the e2e tests with this PR.

File: .ibm/pipelines/value_files/values_showcase-rbac.yaml [hunk 1]
```diff
@@ -255,9 +257,41 @@ upstream:
         # @default -- `quay.io/janus-idp/backstage-showcase:latest`
         image: '{{ include "backstage.image" . }}'
         command:
-          - python
-          - install-dynamic-plugins.py
-          - /dynamic-plugins-root
+          - sh
+          - '-c'
+          - |
+            cat > /rbac-conditions/conditional-policies.yaml <<EOF
```
Comment (subhashkhileri) [inline .ibm/pipelines/value_files/values_showcase-rbac.yaml:263]: @josephca Is it possible to move it into a config map and add it in value-files under `extraVolumeMounts` similar to `.ibm/pipelines/resources/config_map/configmap-rbac-policy-rhdh.yaml`

Comment (josephca) [reply .ibm/pipelines/value_files/values_showcase-rbac.yaml:263]: @subhashkhileri That's possible. However, if we move it into a config map, it won't allow modifying the conditions on the file by making the file 'readable'.  I intended to generate the condition file to accommodate any upcoming feature improvements and tests. Thanks!

**Pull request merged**